bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
+        if valid_email_check:
+            result_message = 'Body contains these invalid emails:\n'
+            invalid_emails = False
+            for line in body:
+                for email in re.findall(r'\S*@\S*', line):
+                    email_check = re.match(r'(^[a-zA-Z0-9_.+-]'
+                                           + r'+@[a-zA-Z0-9-]+\.'
+                                           + r'[a-zA-Z0-9-.]+$)', email)
+                    # if eamil do not match regex there is no need to
+                    # look up dns
+                    if email_dns_check and email_check:
+                        email_check = not is_email(email, diagnose=True,
+                                                   check_dns=True).ERROR_CODES
+                    if not email_check:
+                        invalid_emails = True
+                        result_message += ' ' + email + '\n'
```

Comment:
An approach a bit more elegant would be to collect the invalid addresses in a list instead of using the `invalid_emails` flag and building the string. You could check if the list is empty in the end and then use `"\n".join()` to build the string more effectively.